pandas/io/formats/html.py

```diff
@@ -74,9 +77,19 @@ def _write_cell(self, s, kind='td', indent=0, tags=None):
                                ('>', r'&gt;')])
         else:
             esc = {}
+
         rs = pprint_thing(s, escape_chars=esc).strip()
-        self.write(u'{start}{rs}</{kind}>'
-                   .format(start=start_tag, rs=rs, kind=kind), indent)
+
+        if self.urls_as_links and _is_url(rs):
+            rs_unescaped = pprint_thing(s, escape_chars={}).strip()
+            start_tag += '<a href="{url}">'.format(url=rs_unescaped)
```

Comment:
Might just be a personal preference but probably better to add `target="_blank"` so that following a link doesn't potential kill a notebook session

Reply:
@WillAyd : You're not alone in that preference 🙂